src/components/Router/Flow.js

```diff
@@ -0,0 +1,91 @@
+import { h, Component } from 'preact'
+import createHistory from 'history/createBrowserHistory'
+
+import { events } from '../../core'
+import {sendScreen} from '../../Tracker'
+import {wrapArray} from '../utils/array'
+
+const history = createHistory()
+
+class Flow extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      step: this.props.step,
+      componentsList: this.props.componentsList
```

Comment:
doesn't seem like this is needed. this state property is always equal to props property right?